tests/Agent/UnitTests/Core.UnitTest/NewRelic.Agent.Core.FromLegacy/LoggerBootstrapperTest.cs

```diff
@@ -111,6 +147,119 @@ public static void Config_IsConsoleEnabled_for_config_is_false_when_not_added_to
             Assert.IsFalse(config.Console);
         }
 
+        [Test]
+        public static void Fatal_exception_can_be_recorded()
+        {
+            Assert.IsFalse(Log.FileLoggingHasFailed);
+            Log.FileLoggingHasFailed = true;
+            Assert.IsTrue(Log.FileLoggingHasFailed);
+        }
+
+        [Test]
+        public void test_ways_to_disable_logging()
+        {
+            ILogConfig config;
+
+            config = GetLogConfig("debug");
+            Assert.IsTrue(config.Enabled);
+
+            config = LogConfigFixtureWithLogEnabled(false);
+            Assert.IsFalse(config.Enabled);
+
+            SetEnvironmentVar("NEW_RELIC_LOG_ENABLED", "0");
```

Comment:
This test could be a lot more concise with the use of `[TestCase()]`

Reply:
Good idea, done.